editor/css/editor-libs/codemirror-override.css

```
@@ -25,6 +25,15 @@
   border-color: var(--text-primary);
 }
 
+.cm-editor .cm-line::selection,
+.cm-editor .cm-line ::selection {
+  background-color: var(--selection-backgroud-color) !important;
```

```suggestion
  background-color: var(--selection-background-color) !important;
```

...and the corresponding changes where this is referenced

---

Thank you, it's fixed now.